src/components/Checkbox/Checkbox.tsx

```diff
@@ -39,67 +50,95 @@ export const onChangeDefault = () => {
   return;
 };
 
+function CheckboxIcon(props) {
+  const { isIndeterminate, isChecked, ...rest } = props;
+
+  const d = isIndeterminate
+    ? "M12,0A12,12,0,1,0,24,12,12.013,12.013,0,0,0,12,0Zm0,19a1.5,1.5,0,1,1,1.5-1.5A1.5,1.5,0,0,1,12,19Zm1.6-6.08a1,1,0,0,0-.6.917,1,1,0,1,1-2,0,3,3,0,0,1,1.8-2.75A2,2,0,1,0,10,9.255a1,1,0,1,1-2,0,4,4,0,1,1,5.6,3.666Z"
+    : "M8.795 15.875l-4.17-4.17-1.42 1.41 5.59 5.59 12-12-1.41-1.41-10.59 10.58z";
+
+  /*
+    boxSize - controls the size of the "checked" icon.
+    color - does absolutely nothing?
+    path:fill - can only control color with hardcoded value?
+  */
+  return (
+    <Icon viewBox="0 0 24 24" boxSize={6} color="ui.focus" {...rest}>
```

Comment:
I'm a bit wary of using Chakra's `Icon` component rather than the DS's `Icon` component.  Is it possible to use the DS one or is this for ease of use?

Reply:
- Changed the checkbox to match DS: 22x22; no longer using the Chakra boxSize
- I took a run at swapping out the Chakra `Icon` component with the DS version and was unsuccessful (checkbox no longer worked properly). Looking at the docs, it seems like there are [specific options for using custom icons](https://chakra-ui.com/docs/media-and-icons/icon), but I did not see a way to use a separate `Icon` component completely. Correct me if I'm wrong and I'll take another stab at it.